tests/src/test_cases/widgets/test_chart.c

```diff
@@ -1,4 +1,4 @@
-#if LV_BUILD_TEST
+#if LV_BUILD_TEST || 1
```

Comment:
Why add `|| 1` ?

Reply:
For debugging :slightly_smiling_face: Removed.